index.d.ts

```diff
@@ -2300,9 +2301,9 @@ declare namespace Eris {
     reactions: { [s: string]: { count: number; me: boolean } };
     referencedMessage?: Message | null;
     roleMentions: string[];
+    stickerItems?: StickerItems[];
```

Comment:
Idk how I didn't notice this was out of order when I made the PR for stickerItems, but I believe it's also alphabetized wrong in the Message structure too.